### DISCUSSION TITLE
Datadog statsd for Airflow

### DISCUSSION THREAD

**Discussion by marekargalas** [airflow-helm/charts#688] [Questions & Answers]
Hey!

Currently I'm using [epoch8/airflow-exporter](https://github.com/epoch8/airflow-exporter) for exporting metrics to Prometheus. I would like to switch it to Datadog and noticed that there is [integration for that.](https://docs.datadoghq.com/integrations/airflow/?tab=host). Any experience how to roll it out? 

From first look it seems feasible:
- there is `pip install 'apache-airflow[statsd]'`  - could that be injected to [extraPipPackages](https://github.com/airflow-helm/charts/blob/1865722b66889c4a18cd3525cd6f4f7a2b89f26e/charts/airflow/values.yaml#L210)? 
- need set a few variables in config (eg [stats_on](https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#statsd-on)) - could that be part of this helm chart `values.yaml` as well?
- not sure what to do with `datadog.yaml` file (I already got Datadog agents running) but they are out of my impact

Any experience or advice is welcome! Definitely I will test on staging one, but just want to double check if that's feasible at all. Thanks!

**Comment by thesuperzapper** [comment]
@marekargalas I am interested if you have solved this problem.

If you have, could you share what you did, and possibly help us add a new FAQ page about using DataDog?
